### ISSUE TITLE
Cannot find the correct path

### ISSUE THREAD

**Issue by xyztlmercury** [yeastrc/proxl-import-plink2#2]
Hi, 

I was trying to convert Kojak output and pLink2 output into the proxl input and ran into several problems:

1. 
This is the command I ran: java -jar kojak2ProxlXML.jar 
And below is the error message:
Exception in thread "main" java.lang.ClassNotFoundException: org.yeastrc.proxl.proxl_gen_import_xml_kojak.program_default.GenImportXMLFromKojakDataDefaultMainProgram
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:56)

2. I encountered problem 1 on my Mac so I switched to a windows system... I was able to execute the .jar file but run into the error below:

>java -jar plink2toProxlXML.jar -p c:\MassSpecData\rawFiles\CX\EcoliRibosome\20180220\pLink_task_2018.09.16.20.06.00\pLink_task_2018.09.16.20.06.00.plink -o c:\MassSpecData\rawFiles\CX\EcoliRibosome\20180220\pLink_task_2018.09.16.20.06.00\pLink_task_2018.09.16.20.06.00.proxl.xml -f c:\MassSpecData\rawFiles\CX\EcoliRibosome\20180220\ip2_6410_10256.fasta
pLink 2.x to proxl XML converter
Author: Michael Riffle <mriffle@uw.edu>
See: https://github.com/yeastrc/proxl-import-plink2
Version: 1.0.7

Loading search parameters... Done.
Loading search results...

Encountered an error during conversion:
can not find data directory: C:\MassSpecData\rawFiles\EcoliRibosome\20180220\pLink_task_2018.09.16.20.06.00\reports

So basically the path where the converter was looking for the reports were not the path that I input (missing "\CX")

3. When I rerun plink, plink generates a new .plink file, and in that case I won't be able to locate and visualize previous search result

I am wondering if you could help me with these.

Thank you very much and looking forward to your reply!

Sincerely,
Xiaorui

**Comment by mriffle** [comment]
Xiaorui, thanks for your issue report.

Regarding the Kojak converter on OSX, we'll look into that separately and get back to you.

Regarding the pLink2 converter:

By default, the converter looks in the .plink file (specified with the -p parameter) for the location of the directory that contains the results.  In this case, your .plink file is saying the data were output to "C:\MassSpecData\rawFiles\EcoliRibosome\20180220\pLink_task_2018.09.16.20.06.00\reports", which is is not the directory you want.

The converter contains an optional parameter, -r, which lets you override this default behavior and specify the actual directory that contains the results. Can you try running the converter also with the -r parameter specifying the reports directory containing your data? Possibly ' -r C:\MassSpecData\rawFiles\CX\EcoliRibosome\20180220\pLink_task_2018.09.16.20.06.00\reports'
